workers/ohsome_quality_analyst/api.py

```diff
@@ -56,26 +30,74 @@ async def load_bpolys(bpolys: str) -> None:
 class IndicatorParameters(BaseModel):
     bpolys: Optional[str] = None
     dataset: Optional[str] = None
-    featureId: Optional[Union[int, str]] = None
+    featureId: Union[int, str, None] = None
     layerName: Optional[str] = None
     fidField: Optional[str] = None
 
 
 class ReportParameters(BaseModel):
     bpolys: Optional[str] = None
     dataset: Optional[str] = None
-    featureId: Optional[Union[int, str]] = None
+    featureId: Union[int, str, None] = None
     fidField: Optional[str] = None
 
 
+def empty_api_response() -> dict:
+    return {
+        "attribution": {
+            "url": "https://ohsome.org/copyrights",
+            "text": "© OpenStreetMap contributors",
+        },
+        "apiVersion": oqt_version,
+    }
+
+
+def int_or_str_param_type(value: str) -> Union[int, str]:
+    """Parse parameter as Interger otherwise parse as String"""
+    try:
+        return int(value)
+    except (ValueError, TypeError):
+        if isinstance(value, str):
+            return value
+        else:
+            raise ValueError("Given parameter is not a valid integer or string")
+
+
+async def load_bpolys(bpolys: str) -> Feature:
+    """Load as GeoJSON object, validate and check size of bpolys"""
+    # TODO: Return API response with error message
+
+    async def check_geom_size(geom):
+        if await db_client.get_area_of_bpolys(geom) > GEOM_SIZE_LIMIT:
+            raise ValueError(
+                "Input GeoJSON Object is too big. "
+                + "The area should be less than {0} sqkm.".format(GEOM_SIZE_LIMIT)
+            )
+
+    bpolys = geojson.loads(bpolys)
+
+    if bpolys.is_valid is False:
+        raise ValueError("Input geometry is not valid")
+    elif isinstance(bpolys, Feature):
+        await check_geom_size(bpolys.geometry)
+        return bpolys
+    elif isinstance(bpolys, (Polygon, MultiPolygon)):
+        await check_geom_size(bpolys)
+        return Feature(geometry=bpolys)
+    else:
+        raise ValueError(
+            "Input GeoJSON Objects have to be of type Feature, Polygon or MultiPolygon"
+        )
```

Comment:
Why do we disallow `FeatureCollection` here? Wouldn't it be sufficient to check if we have multiple Features?

Reply:
In contrast to the CLI there exists no logic to handle FeatureCollection in the API. I want to address this separately after geo_interface for indicators/reports is implemented.

So for now FeatureCollection is not allowed as input using the API.

Reply:
Okay I think you misread me here. In the past we allowed people to give OQT a GeoJSON with a FeatureCollection with **one** `Feature`. I keeping this could be fairly easy for API and CLI. It could be something like this:
```python
elif isinstance(bpolys, FeatureCollection):
    if len(bpolys.features) == 1:
        # either this one to use it recursive, or duplicate code by copying the Feature implementation
        return load_bpolys(geojson.drops(bpolys.features[0]))
    else:
        raise ValueError("Only one Feature is supported in a FeatureCollection")
```

Reply:
General question? Why is this implemented in `api.py` and not in `oqt.py`? Shouldn't the code be similar for both the API and CLI?

Reply:
I added support for FeatureCollection with one Feature as input to API.

Reply:
The goal is to share as much logic between API and CLI as possible as well as similar input/output. I hope to address this in future PRs. Right now the CLI supports GeoJSON FeatureCollection with more than one Feature. But as I said I hope to unify both of them in the near future.